### ISSUE TITLE
Multiple sensors stuck on "vibrating"

### ISSUE THREAD

**Issue by pilsnerz** [Shmoopty/rpi-appliance-monitor#37]
So I've gotten everything to work with 1 sensor on my dryer but now having issues when connecting another sensor. I've followed directions with a new settings file set to a unique GPIO which I've set to 14 and 15 (in separate files).  Also added a second line to the rc.local file like below. They both seem to work and will get a message the washer or dryer has started but now I get no end notification. When I SSH in they seem to be just stuck on "vibrating" even though they are not. Not sure if my rc.local file is wrong? Do i need multiple vibration.py files? 

python /home/pi/vibration.py /home/pi/vibration_settings.ini &

python /home/pi/vibration.py /home/pi/vibration_settingsW.ini &

**Comment by Shmoopty** [comment]
Everything you describe here is correct. Pins 14 and 15 should work.  You don't need multiple copies of the program.

I would recommend trying to run each of those commands separately from the command line so you can see the output.

Good luck, and keep us posted!

**Comment by pilsnerz** [comment]
Thanks for the response. So I tried them each separately and they worked fine except for giving me 2 messages each but I think because I was going into putty and running the command "sudo /etc/rc.local" to start seeing the output and the program was already running. I ended up switching the start up file to have GPIO 15 config first and its working fine now!

Maybe it was doing something weird because I was testing using that command...I'm an idiot with python and programming. Is there a way to start seeing the output when SSHing in? Anyway this has been a fun learning experience, thank you!
